merkletree/pad.go

```diff
@@ -5,6 +5,7 @@ import (
 	"errors"
 
 	"github.com/coniks-sys/coniks-go/crypto"
+	conikshasher "github.com/coniks-sys/coniks-go/crypto/hasher/coniks"
```

Comment:
Ideally, this package should be imported using a blank import name. I used `conikshasher` for now because we import both `hasher` and `hasher/coniks` sometimes.